net-proxy/mitmproxy/mitmproxy-5.1.1.ebuild

```diff
@@ -21,18 +21,21 @@ RDEPEND="
 	>=dev-python/certifi-2015.11.20.1[${PYTHON_USEDEP}]
 	>=dev-python/click-6.2[${PYTHON_USEDEP}]
 	>=dev-python/cryptography-2.1.4[${PYTHON_USEDEP}]
+	>=dev-python/flask-1.0.4[${PYTHON_USEDEP}]
 	>=dev-python/hyper-h2-3.0.1[${PYTHON_USEDEP}]
 	>=dev-python/hyperframe-5.1.0[${PYTHON_USEDEP}]
 	>=dev-python/kaitaistruct-0.7[${PYTHON_USEDEP}]
 	>=dev-python/ldap3-2.5[${PYTHON_USEDEP}]
 	>=dev-python/passlib-1.6.5[${PYTHON_USEDEP}]
 	>=dev-python/protobuf-python-3.6.0[${PYTHON_USEDEP}]
+	>=dev-python/publicsuffix-2.20190205[${PYTHON_USEDEP}]
 	>=dev-python/pyasn1-0.3.1[${PYTHON_USEDEP}]
 	>=dev-python/pyopenssl-17.5[${PYTHON_USEDEP}]
 	>=dev-python/pyparsing-2.1.3[${PYTHON_USEDEP}]
 	>=dev-python/pyperclip-1.6.0[${PYTHON_USEDEP}]
 	>=dev-python/requests-2.9.1[${PYTHON_USEDEP}]
 	>=dev-python/ruamel-yaml-0.15[${PYTHON_USEDEP}]
+	>=dev-python/ruamel-yaml-clib-0.1.2[${PYTHON_USEDEP}]
```

Comment:
why this dependency?

Reply:
Removed as it is included with ruamel-yaml